scripts/src/java/org/oppia/android/scripts/common/BazelClient.kt

```diff
@@ -130,6 +131,59 @@ class BazelClient(private val rootDirectory: File, private val commandExecutor:
     return correctedTargets
   }
 
+  /**
+   * Runs code coverage for the specified Bazel test target
+   *
+   * @param bazelTestTarget Bazel test target for which code coverage will be run
+   * @return the generated coverage data as a list of strings
+   * or null if the coverage data file could not be parsed
+   *
+   * Null return typically occurs when the coverage command fails to generate the 'coverage.dat' file
+   * This can happen due to: Test failures or misconfigurations that prevent the coverage data
+   * from being generated properly
+   */
+  fun runCoverageForTestTarget(bazelTestTarget: String): List<String>? {
+    val coverageData = executeBazelCommand(
```

Comment:
'coverageCommandOutputLines' would be a much clearer name. As-is, it's currently confusing whether this contains the command's output lines or the actual coverage dat file contents.

Reply:
Yeah! Changed it to `coverageCommandOutputLines` throughout the implementation. Thanks!